satpy/etc/enhancements/generic.yaml

```
@@ -3,9 +3,13 @@
 enhancements:
   default:
     operations:
-    - name: stretch
-      method: !!python/name:satpy.enhancements.contrast.stretch
-      kwargs: {stretch: linear}
+      - name: warn_no_predefined
+        method: !!python/name:satpy.enhancements.contrast.noop_with_warning
+        kwargs:
+          msg: "No YAML enhancement configured for {name!r}. Falling back to a dynamic linear stretch."
```

Adding a link to documentation, and changing `configured` to `found`.  It is possible that the user has tried to configure an enhancement, but that they did something wrong and the enhancement wasn't found.  Also, the warning message is factually incorrect for non-floating data, because it is only falling back to a dynamic linear stretch for floating data.  For integer data no enhancement is defined.

```suggestion
          msg: "No YAML enhancement found for {name!r}. Falling back to a dynamic linear stretch in case of floating point data, or no enhancement otherwise. See https://satpy.readthedocs.io/en/stable/enhancements.html for documentation on defining an enhancement."
```